index.html

```diff
@@ -3,20 +3,76 @@
 <head>
 </head>
 <body>
-    
-    <!-- Eyram -->
-    <div id="HERO_BG">
-        <div id="HEADER"></div>
-        <div id="HERO"></div>
-    </div>
-    <div id="TOGETHER"></div>
-    <div id="CONVERSATIONS"></div>
-    <!-- Joe -->
-    <div id="USER"></div>
-    <div id="Rectangle"></div>
-    <div id="CTA"></div>
-    <div id="Rectangle_Copy">
-        <div id="CTB"></div>
-    </div>    
+     <header>
+      <img src="file:///C:/Users/jenni/huddle/images/bg-hero-mobile.svg">
+      <div class="header_content1">
+        <div class = "header_logo">
+          <img src="file:///C:/Users/jenni/huddle/images/logo.svg">
+        </div>
+        <div class = "header_text">
+          <p>Try It Free</p>
+        </div>
+       </div>  
+       <div class="header_content2">
+        <div class="header_item1">
+          <h1>Build The Community Your Fans Will Love</h1>
+          <p>Huddle re-imagines the way we build communities. You have a voice, but so does your audience. Create connections with your users as you engage in genuine discussion. </p>
+          <button>Get Started For Free</button>
+        </div>
+        <div class="header_item2">
+          <img src="file:///C:/Users/jenni/huddle/images/illustration-mockups.svg">
```

Comment:
please use relative paths for the images
I won't have "file:///C:/Users/jenni/huddle/images/illustration-mockups.svg" on my pc. 

use "./images//illustration-mockups.svg" that way it will work on everyone's pc